bentoml/adapters/annotated_image_input.py

```diff
@@ -113,6 +113,23 @@ def predict(
             <input type="submit">
         </form>
 
+    OR by python requests:
+
+    .. code-block:: python
+
+        import requests
+
+        with open("test.jpg", "rb") as f:
+            image_bytes = f.read()
+        with open("anno.json", "rb") as f:
```

Comment:
give the user an alternative to use some in-memory data as the JSON annotation data?

e.g.
```
annotations = { "age": 10, "bar": "foo" }
anno_bytes = json.dumps(annotations).encode()
```

Reply:
That is definitely helpful! I think in many cases the annotation part of the request is created at runtime and not read from file.